app/models/user.rb

```diff
@@ -0,0 +1,3 @@
+class User < ActiveRecord::Base
+  validates :username, :token, presence: true
```

Comment:
Lets also validate the presence of `email`, I think we need to make user we have user's email in order to be sure we will be able to contact them about the gem adoption process when necessary.